requirements.txt

```diff
@@ -71,3 +71,4 @@ urllib3==1.15.1
 whitenoise==2.0.4
 smart_open==1.5.3
 unicodecsv
+git+https://github.com/hashknot/django-pg-fts
```

Comment:
do you mind moving this to the diging organization? I'd like to keep everything together in case further changes are necessary down the road.

Reply:
I had thought of asking you how should I do this, completely slipped my mind.

Reply:
I don't have permissions to fork to diging.

Reply:
hm, can you transfer the repo to me and I'll transfer it to diging?